contracts/test/EVMPrecompileTester.js

```diff
@@ -0,0 +1,190 @@
+    
+                // Get a contract instance
+                erc20 = new ethers.Contract(contractAddress, contractABI, signer);
+                console.log("end of before");
+            });
+    
+            it("Transfer function", async function() {
```

Comment:
can we do a transfer from an address with no token balance and make sure it fails

Reply:
Updated